webapp/client/src/lib/propTypes.js

```diff
@@ -47,6 +47,16 @@ export const selectionFieldPropType = PropTypes.exact({
   errors: PropTypes.arrayOf(PropTypes.string),
 });
 
+export const stepHeaderPropType = PropTypes.exact({
+  title: PropTypes.string,
+  intro: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.element,
+    PropTypes.object,
+  ]),
+  hideIntro: PropTypes.bool,
```

Comment:
Shouldn't `hideIntro` and `intro` be optional?

Reply:
I thought it would be optional because we do not use isRequired.

This is basically a copy of the PropTypes.exact that we use at some places to describe the stepHeader